protocol-configuration/proposalsConfig.ts

```diff
@@ -1,13 +1,21 @@
 import { ProposalCategory, TemplatedProposalsConfigMap } from '@custom-types/types';
 
-import fip_x from '@proposals/description/fip_x';
-
 import tip_118 from '@proposals/description/tip_118';
+import pcv_guardian_v3 from '@proposals/description/pcv_guardian_v3';
 
 // This config contains each of the active governance proposals in the protocol - DAO, TC, or otherwise.
 // The key for each should be the proposal name or number, and the value should be the TemplatedProposalConfig.
 
 export const ProposalsConfig: TemplatedProposalsConfigMap = {
+  pcv_guardian_v3: {
+    deploy: true,
+    totalValue: 0,
+    proposal: pcv_guardian_v3,
+    proposalId: '',
+    affectedContractSignoff: [],
+    deprecatedContractSignoff: [],
+    category: ProposalCategory.DEBUG
```

Comment:
Will want to change this proposal category to TC I imagine?

Reply:
yep, good catch

Reply:
actually has to be submitted to DAO because it revokes & grant Guardian & PCV Controller